humble/README.md

```diff
@@ -59,7 +59,7 @@ They also show clear trade-offs between synchronous and asynchronous publishing
 In [Section 2](#-2-rep-2004-code-quality-metrics), adherence to [REP-2004](https://www.ros.org/reps/rep-2004.html) is compared.
 The implementations are quite similar, despite some inconsistencies in the reporting due to differences in self reporting.
 Under most of the categories for the parts that are documented each implementation are comparable.
-Despite this there is an appreciable difference as Cyclone DDS is currently declared as quality level 2, and Fast RTPS is rated as quality level 1.
+Cyclone DDS is currently declared as quality level 2 and Fast RTPS is declared as quality level 1.
```

Comment:
Let's not get rid of this, as there is a difference.  If you don't like the wording, I'd also be OK with:

```suggestion
Despite this there is a difference as Cyclone DDS is currently declared as quality level 2 and Fast RTPS is declared as quality level 1.
```